# CVE-2017-18207


## Metadata
| Published | Last Modified | Assigner | Version | Format |
| --------- | ------------- | -------- | ------- | ------ |
| 2018-03-01 05:29:00 | 2020-01-21 22:15:00 | cve@mitre.org | 4.0 | MITRE | 

## Description
| Language | Description |
| -------- | ----------- |
| en | "** DISPUTED ** The Wave_read._read_fmt_chunk function in Lib/wave.py in Python through 3.6.4 does not ensure a nonzero channel value, which allows attackers to cause a denial of service (divide-by-zero and exception) via a crafted wav format audio file. NOTE: the vendor disputes this issue because Python applications ""need to be prepared to handle a wide variety of exceptions.""" |

## Configurations
| Vulnerable | CPE 23 URI | CPE Name | Operator |
| ---------- | ---------- | -------- | -------- |
| True | cpe:2.3:a:python:python:*:*:*:*:*:*:*:* | 3.6.4 | [] | OR |

## CVSSv3 Information
| Score | Severity | Impact Score | Exploitability Score | Vector String | Version | Attack Vector | Attack Complexity | Privileges Required | User Interaction | Scope | Confidentiality Impact | Integrity Impact | Availability Impact |
| ----- | -------- | ------------ | -------------------- | ------------- | ------- | ------------- | ----------------- | ------------------- | ---------------- | ----- | ---------------------- | ---------------- | ------------------- |
| 6.5 | MEDIUM | 3.6 | 2.8 | CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H | 3.0 | NETWORK | LOW | NONE | REQUIRED | UNCHANGED | NONE | NONE | HIGH |

## CVSSv2 Information
| Score | Severity | Impact Score | Exploitability Score | Vector String | Version | Access Vector | Access Complexity | Authentication | User Interaction Required | Obtain All Privilge | Obtain User Privilege | Obtain Other Privilege | Confidentiality Impact | Integrity Impact | Availability Impact |
| ----- | -------- | ------------ | -------------------- | ------------- | ------- | ------------- | ----------------- | -------------- | ------------------------- | ------------------- | --------------------- | ---------------------- | ---------------------- | ---------------- | ------------------- |
| 4.3 | MEDIUM | 2.9 | 8.6 | AV:N/AC:M/Au:N/C:N/I:N/A:P | 2.0 | NETWORK | MEDIUM | NONE | 1 | False | False | False | NONE | NONE | PARTIAL |

## Problem Type
CWE-369

## References
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
https://bugs.python.org/issue32056|https://bugs.python.org/issue32056|MISC|['Issue Tracking', 'Vendor Advisory']
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
openSUSE-SU-2020:0086|http://lists.opensuse.org/opensuse-security-announce/2020-01/msg00040.html|SUSE|[]
